CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.12.3] - 2024-07-04
+
+- Fix SQL query that finds alerts for current user.
```

Comment:
You are missing the "###fixed" block.

Also I personally dislike technical comments in the changelog.
I think the description should be understandable by the users of the plugin which may not be developers.

Reply:
:+1: